packages/vm/src/calls.rs

```diff
@@ -745,8 +748,10 @@ mod tests {
             let mut instance = mock_instance(&CONTRACT, &[]);
             setup(&mut instance, CHANNEL_ID, ACCOUNT);
             let packet = mock_ibc_packet_ack(CHANNEL_ID, br#"{}"#).unwrap();
-            let ack = IbcAcknowledgement {
-                acknowledgement: br#"{}"#.into(),
+            let ack = IbcAcknowledgementWithPacket {
+                acknowledgement: IbcAcknowledgement {
+                    data: br#"{}"#.into(),
+                },
```

Comment:
This can now be simplified to

```suggestion
                acknowledgement: IbcAcknowledgement::new(br#"{}"#),
```